src/main/webapp/WEB-INF/view/notifications.jsp

```diff
@@ -16,7 +16,7 @@
   <link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
-  <p> Hello! This is your notifcation page</p>
+  <h4> Back from fishing? This is what you missed:</h4>
```

Comment:
Let's make this smaller. `<p>` is actually fine. If you can add a 20px margin to it it'll also make it cleaner.